src/lib/logger.ts

```diff
@@ -0,0 +1,62 @@
+import { format, transports, createLogger, Logger } from 'winston';
+import DailyRotateFile from "winston-daily-rotate-file";
+
+import loggerConfig from '../../logger.config.json'
+
+const { combine, timestamp, printf, colorize } = format;
+
+function getTransporters(){
+    const dailyRotateFileTransport = getTransportersFromConfig(loggerConfig);
+    return [
+        new transports.Console(),
+        ...dailyRotateFileTransport
+      ]
+}
+
+function getTransportersFromConfig(config:any){
+    return config.transporters.map(x => {
+        if(x.transporterType === 'file'){
+            return new DailyRotateFile(x.options)
+        }
+    }).filter(x => x)
+}
+
+function getFormat(){
+    return combine(
+        ...getFormatFromConfig(loggerConfig)
+    )
+}
+
+function getFormatFromConfig(config:any){
+    return config.format.map(x => {
+        if(x === "colorize"){
```

Comment:
use

if(x in ['coliorize','timestamp']
{
  return eval(x)();
}

Reply:
is eval not python specific?